### PR TITLE
HRINFO-594 powered by styling

### PR DESCRIPTION
**Pull request by amyvbenson**

### CODE REVIEW

**Comment by attiks** [comment]
:+1: from me, @guillaumev can you check as well and merge if all ok

File: sites/all/modules/ar/ar_assessments/js/ar_assessments.js [hunk 1]
```diff
@@ -405,7 +405,7 @@ Drupal.behaviors.arAssessmentsAssessments = {
       },
 
       defaultRoute: function (actions) {
-        this.navigate('list/1', {trigger: true});
```
Comment (guillaumev) [inline sites/all/modules/ar/ar_assessments/js/ar_assessments.js]: @amyvbenson Unless this changed recently, the default route should normally be the list view

Comment (amyvbenson) [inline sites/all/modules/ar/ar_assessments/js/ar_assessments.js]: @guillaumev it was decided on https://humanitarian.atlassian.net/browse/HRINFO-594 to try just showing the table view to make it consistent with the other pages so I've changed the default route and removed the tabs.

Comment (guillaumev) [inline sites/all/modules/ar/ar_assessments/js/ar_assessments.js]: @amyvbenson Ok on my side, here is the case where I was told to show the list view as default: https://humanitarian.atlassian.net/browse/HRINFO-587 CC @emmajane

Comment (amyvbenson) [inline sites/all/modules/ar/ar_assessments/js/ar_assessments.js]: @guillaumev yep, the discussion on HRINFO-594 came after that ticket. The table view is now a lot tidier so we're going to try it out with just that, but will be easy to revert if people prefer the list view.

Comment (guillaumev) [inline sites/all/modules/ar/ar_assessments/js/ar_assessments.js]: Ok then feel free to go ahead and merge it.

**Pull request merged**